src/actions/users.js

```diff
@@ -559,14 +547,20 @@ export function updateMyTermsOfServiceStatus(termsOfServiceId: string, accepted:
         }));
         const {data, error} = response;
         if (data) {
-            dispatch({
-                type: UserTypes.RECEIVED_TERMS_OF_SERVICE_STATUS,
-                data: {
-                    create_at: new Date().getTime(),
-                    terms_of_service_id: accepted ? termsOfServiceId : null,
-                    user_id: getCurrentUserId(getState()),
-                },
-            });
+            if (accepted) {
+                const user = {...getCurrentUser(getState())};
+                user.terms_of_service_id = termsOfServiceId;
+                user.terms_of_service_create_at = new Date().getTime();
```

Comment:
I'm not following why we need this here, is not being used anywhere

Reply:
You're right, it's not being used. Removed.